test/exitHandler.js

```diff
@@ -147,6 +147,33 @@ contract('ExitHandler', (accounts) => {
         assert(aliceBalanceBefore.add(new BN(50)).eq(aliceBalanceAfter));
       });
 
+      it('Should allow to limbo exit', async () => {
+        const period = await submitNewPeriod(depositTx);
+        const depositProof = period.proof(depositTx);
+        const outputIndex = 1;
+        const exitStake = exitHandler.exitStake();
+        
+        // transaction hash + constant signed by alice
+        const signature = (transferTx + c).sign(alicePriv);
+
+        const inTxData = Tx.parseToParams(transferTx);
```

Comment:
We can't calculate transferProof because period is not getting submitted(transferTx is inflight) so i am parsing it to data and then passing it to function and in our contract we can parse this data to obtain the transaction.